tools/src/main/scala/scala/scalanative/build/LLVM.scala

```diff
@@ -150,6 +150,16 @@ private[scalanative] object LLVM {
     copyOutput(config, buildPath)
   }
 
+  def dsymutil(config: Config, path: Path) = {
+    val dsymutil = Discover.discover("dsymutil", "LLVM_BIN")
```

Comment:
Currently `discover` is always failing if it cannot find given binary. Maybe let's introduce an additional variant of it `def tryDiscover(String, String): Option[Path]`. If we won't find it should not be a fatal error, but warning instead. It would be great if in case when we don't find dsymutil we point to the user that we couldn't find this program, possibly show the paths we tried, and point him up that he can use the `LLVM_BIN` env to try fix this issue.